### PR TITLE
Add SwitchEncryptionOpts Nemesis

### PR DESCRIPTION
**Pull request by amoskong**
## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle)
- [x] I gave variables/functions meaningful self-explanatory names
- [x] I didn't leave commented-out/debugging code
- [x] I didn't copy-paste code
- [x] I added the relevant `backport` labels
- [x] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] ~~I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)~~
- [ ] ~~All new and existing unit tests passed (`hydra unit-tests`)~~
- [x] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: sdcm/cluster.py [hunk 1]
```diff
@@ -2346,6 +2346,13 @@ def get_test_keyspaces(self):
         out = self.run_cqlsh(node=self.nodes[0], cql_cmd='select keyspace_name from system_schema.keyspaces')
         return [ks.strip() for ks in out.split('\n')[3:-3] if 'system' not in ks]
 
+    def get_test_tables(self, keyspace=None):
```
Comment (roydahan) [inline sdcm/cluster.py]: I think we should have something like this already.

Comment (amoskong) [reply sdcm/cluster.py]: I only found get_test_keyspaces()

**Comment by fruch** [comment]
@amoskong  
please install precommit hock on your working direcotry

see:
https://github.com/scylladb/scylla-cluster-tests#contribution

**Comment by bentsi** [comment]
@amoskong
1. You have a redundant space inside the precommit checklist.
You shoud create a PR and then select the checks or put [x] without spaces
2. Please fix CI issues

**Comment by fruch** [comment]
@amoskong is this PR still relevant ?

**Comment by amoskong** [comment]
> @amoskong is this PR still relevant ?

I had prepared a V2, will update the PR after testing.

**Comment by fruch** [comment]
@amoskong a month passed, can we close this one ? and reopen/open a new PR once ready ?

**Comment by amoskong** [comment]
> @amoskong a month passed, can we close this one ? and reopen/open a new PR once ready ?

Ok, I split this PR to multiple PRs.
Let's close this one, Thanks.

**Pull request closed without merging**